drivers/gles3/shader_gles3.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "shader_gles3.h"
+#include "core/variant/variant.h"
```

Comment:
The include should go further down where the other includes are. But why do you need Variant?

Reply:
Looks like it is to access PackedInt32Array